src/main/java/com/epam/ta/reportportal/ws/model/StartRQ.java

```diff
@@ -39,8 +38,7 @@
 @JsonInclude(Include.NON_NULL)
 public class StartRQ {
 
-	@NotNull
-	@NotEmpty
+	@NotBlank
```

Comment:
https://github.com/hibernate/hibernate-validator/blob/b173247ab36545b90f2bf64a3838f39f4aab2eb1/engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/NotBlankValidator.java#L33

Do we want to allow null values everywhere where NotEmpty and NotNull were?

Reply:
I checked with null and values. It works properly